packages/wrangler/src/api/startDevWorker/ProxyController.ts

```diff
@@ -362,7 +367,7 @@ export class ProxyController extends EventEmitter {
 	}
 	onReloadComplete(data: ReloadCompleteEvent) {
 		this.latestConfig = data.config;
-		this.latestBundle = data.bundle;
+		this.latestBundle = data.bundle; // TODO(now): update `latestBundle` type
```

Comment:
- [ ] update `latestBundle` type and remove comment

Reply:
I've removed this comment—we might want to update this type in the future, but we can address that if needed. It's definitely not a `TODO(now)` urgency refactoring

Reply:
https://github.com/cloudflare/workers-sdk/pull/4450/commits/44c727e39c8560cc960f720529505a393ff2de53